### DISCUSSION TITLE
Can collision block of EPOCH discribe the excited state of atom?

### DISCUSSION THREAD

**Discussion by keithbennett** [epochpic/epoch#131] [General]
In GitLab by jiansheng on 18 Nov 2020, 11:07 UTC (GitLab issue GL#2277)

Hi:
   I'm a beginner, and I start to learn EPOCH from testing many different benchmarks. And I found the excited state of atoms couldn't be described in EPOCH.  In general, there are three states when energetic particles come across atoms: elastic collision, ionisation, and excited state. But, When I read manual, I found there are only first two functions in collision block of EPOCH. So, how to descibe excited state of atoms in EPOCH?

**Comment by keithbennett** [comment]
In GitLab by jiansheng on 18 Nov 2020, 11:07 UTC

changed due date to April 04, 2023

**Comment by ZippoFight** [comment]
Hi, is elastic collisions considered during Coulomb collision?

**Comment by Status-Mirror** [comment]
Hey @ZippoFight,

The [collisions package](https://epochpic.github.io/documentation/input_deck/input_deck_collisions.html) considers elastic collisions, and collisional ionisation can also be turned on in this input deck block.

From theory, we know how the momenta of two particles change when they collide elastically (binary collision), and this varies with the impact parameter. From the Coulomb logarithm, we know the range of impact parameters an incident particle can interact with. From the time-step, incident particle speed, and target particle number density, we can determine how many collisions occur. The total scatter angle from $n$ binary collisions is known from [Nanbu theory](https://journals.aps.org/pre/abstract/10.1103/PhysRevE.55.4642), and EPOCH samples this scatter angle.

Hope this helps explain what the code is doing,
Stuart

**Reply by ZippoFight** [reply]
Hi, @Status-Mirror , thank you for your reply.

When the collisions between electrons and atoms (include ioinized atoms, i.e. ions) are analyzed, types including "elastic collision", "inelastic collision", "impact ionization", "impact excitation", and "Coulomb collision" are always talked about. Is elastic collision is the same as Coulomb collision? 

If electrons and neutral atoms are collided in Epoch, how is it processed? I suppose Coulomb collision is between charged particles. 

Another question is that if charge exchange collision is possible to be considered.

**Reply by Status-Mirror** [reply]
Yes, at present EPOCH only calculates cumulative Coulomb collisions. Equation 5 in the [Nanbu](https://journals.aps.org/pre/pdf/10.1103/PhysRevE.55.4642) paper describes the scatter angle considered in a collision between two charged particles. EPOCH currently has no support for collisions with neutral particles. We can also model electron impact ionisation (either atom or ion targets), but this is a separate package. We do not consider excitation.

We have no plans to implement a charge exchange collision model at present. Do you know if this has been done before in another code? EPOCH is written in a way where new physics packages can be slotted in.

**Reply by ZippoFight** [reply]
Thanks. As I know WarpX support the process of charge exchange between kinetic particles and background species, but I am not sure if this can be applied in EPOCH.

about the collisions between electron and neutral atom or between two neutral atoms, if the collision modelling is turned on in EPOCH by "collide = electron atom on" and "collide = atom atom  on", with the collisional ionization turned on at the same time by "use_collisional_ionisation = T", what will happen in the calculation?

**Reply by Status-Mirror** [reply]
Sorry Zippo, I missed the second part of this question.

The collide command does not affect collisional ionisation. When you switch collisional ionisation on, the code considers ionisation between all atom/ion species and all electron species, regardless of what collide is set to. Collide only controls cumulative Coulomb scatter between species.

In your example here, you say `collide = atom atom on`, but EPOCH will not attempt to calculate Coulomb collisions between two neutral species (assuming both your atoms are neutral).

**Comment by Status-Mirror** [comment]
And to reply to the first question, EPOCH does not currently calculate the excited states of particles. We assume particles are in the ground state when calculations require knowledge of the electron configuration.
